__tests__/app.test.js

```diff
@@ -42,3 +42,49 @@ describe("/api/topics", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    describe("STATUS 200", () => {
+      test("should respond with an article object with properties: author, title, article_id, body,topic,created_at and votes", () => {
+        const ARTICLE_ID = 1;
+        return request(app)
+          .get(`/api/articles/${ARTICLE_ID}`)
+          .expect(200)
+          .then(({ body: { article } }) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
```

Comment:
This test is not accurate enough, how do you know you are getting exactly the article you expect?

Reply:
I added an extra assertion on that test for it to equal that specific article as per latest commit.